public/tic.js

```diff
@@ -0,0 +1,126 @@
+class Player {
+    constructor(name){
+        this.name = name
+    }
+    cellsChosen = []
+}
+let playerX = new Player('Player X')
+let playerO = new Player('Player O')
+
+let winningPossibilities = [
+    ['1','2','3'],
+    ['1','5','9'],
+    ['1','4','7'],
+    ['2','5','8'],
+    ['3','5','7'],
+    ['3','6','9'],
+    ['4','5','6'],
+    ['7','8','9']
+]
+
+let arr = []
+let gameBoard = document.getElementById("game-board")
+startGame()
+
+function startGame(){
+
+    let playAgain = document.getElementById('playAgain')
+    playAgain.addEventListener('click',()=>{
+        location.reload()
+    })
+
+    this.innerHTML = ''
```

Comment:
`this` here points to the global `window` object. Setting its `innerHTML` to an empty string doesn't do much since it doesn't have access to the DOM.